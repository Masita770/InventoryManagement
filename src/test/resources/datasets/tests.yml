items:
  - items_id:  3
    category:  "机"
    item:  "オフィスデスク"
    registration_date:  "2025-03-06 23:45:18"
    deleted_date:  "0000-00-00 00:00:00"
  - items_id:  4
    category:  "机"
    item:  "キッチンテーブル"
    registration_date:  "2025-03-06 23:45:57"
    deleted_date:  "0000-00-00 00:00:00"
  - items_id:  5
    category:  "机"
    item:  "ゲーミングデスク"
    registration_date:  "2025-03-06 23:46:32"
    deleted_date:  "0000-00-00 00:00:00"