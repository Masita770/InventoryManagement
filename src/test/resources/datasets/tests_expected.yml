items:
  - itemsId: 1
    category:  "机"
    item:  "オフィスデスク"
    registrationDate:  "0000-00-00 00:00:00"
    deletedDate:  "0000-00-00 00:00:00"
  - itemsId: 2
    category:  "机"
    item:  "キッチンテーブル"
    registrationDate:  "0000-00-00 00:00:00"
    deletedDate:  "0000-00-00 00:00:00"
  - itemsId: 3
    category:  "机"
    item:  "ゲーミングデスク"
    registrationDate:  "2024-09-16 22:36:15"
    deletedDate:  "0000-00-00 00:00:00"